agent/command/papertrail_trace.go

```diff
@@ -0,0 +1,156 @@
+package command
+
+import (
+	"context"
+	"crypto/sha256"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/evergreen-ci/evergreen/agent/internal"
+	"github.com/evergreen-ci/evergreen/agent/internal/client"
+	"github.com/evergreen-ci/evergreen/thirdparty"
+	"github.com/evergreen-ci/evergreen/util"
+	"github.com/mitchellh/mapstructure"
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+)
+
+const (
+	productionPapertrailAddress = "https://papertrail.devprod-infra.prod.corp.mongodb.com"
```

Comment:
Would it make sense to have this exist in thirdparty instead? Doesn't seem like something the agent needs to know.

Reply:
I moved this to thirdparty and introduced a new function to handle initializing a client with this address!